mathesar_ui/src/pages/schema/SchemaPage.svelte

```diff
@@ -36,183 +30,81 @@
   // eslint-disable-next-line @typescript-eslint/no-inferrable-types
   export const section: string = 'overview';
 
-  let isCreatingNewTable = false;
-
+  const addEditModal = modal.spawnModalController();
+
+  type TabsKey = 'overview' | 'tables' | 'explorations';
+  type TabItem = { label: string; id: TabsKey };
+  const tabs: TabItem[] = [
+    {
+      label: 'Overview',
+      id: 'overview',
+    },
+    {
+      label: 'Tables',
+      id: 'tables',
+    },
+    {
+      label: 'Explorations',
+      id: 'explorations',
+    },
+  ];
+  let activeTab: TabItem;
```

Comment:
The tab selection should be reflected in the url. There is prop here at line 31 named 'section' which provides the information of the section mentioned on the url. It should be set as the initial selected tab.

Whenever a tab is clicked, it should redirect to their respective urls. Refer SchemaRoute.svelte where the urls are already specified.

Reply:
#1922